docs/usage.rst

```diff
@@ -289,6 +293,16 @@ or in this `serverfault.com answer <https://serverfault.com/a/9717/330351>`_.
 Note that setting the ``REQUESTS_CA_BUNDLE`` or ``CURL_CA_BUNDLE`` environment
 variables influences other programs that use these variables, too.
 
+If you do not know which CA certificate the HMC has been configured with,
+you can use the following OpenSSL commands to display the certificates
+returned by the HMC. Look at the Issuer of the highest certificate in the CA
+chain (usually the last one displayed):
+
+.. code-block:: sh
+
+    $ echo | openssl s_client -showcerts -connect $hmc_ip:6794 2>/dev/null | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' >/tmp/get-server-certs.pem
```

Comment:
minor: when you dump the whole stuff into a pem file anyway, why filter out those two lines?

Reply:
It is not filtering out these two lines, it outputs these two lines and everything in between, i.e. exactly what would be in a .PEM file.